staging/src/k8s.io/client-go/rest/config.go

```diff
@@ -120,6 +120,42 @@ type Config struct {
 	// Version string
 }
 
+var _ fmt.Stringer = new(Config)
+var _ fmt.GoStringer = new(Config)
+
+type sanitizedConfig *Config
+
+type sanitizedAuthConfigPersister struct{ AuthProviderConfigPersister }
+
+func (sanitizedAuthConfigPersister) String() string { return "--- REDACTED ---" }
```

Comment:
needed GoString() as well, I guess... this showed up in the unit test as `AuthConfigPersister:rest.sanitizedAuthConfigPersister{AuthProviderConfigPersister:rest.fakeAuthProviderConfigPersister{}`

Reply:
Done